### PR TITLE
Tanyapina/definition2

### PR DESCRIPTION
**Pull request by TanyaPina**
This code has a second definition.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # Sinuheguerracom2
 
 This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 1.1.1.
+This is a new definition.git. This is Tanya's change
```
Comment (sinuheg) [inline README.md:4]: Let's keep both changes. I am ok with mine going second.

**Review by sinuheg — COMMENTED** [review]
Just let's review the conflicts.

File: README.md (hunk 1 above)
Comment (TanyaPina) [reply README.md:4]: Sounds good, will resolve the conflict.

**Review by sinuheg — APPROVED** [review]
Looks good. Ship it!

**Pull request merged**